kube/file-vault/file-vault-deployment.yml

```diff
@@ -0,0 +1,207 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+  name: file-vault
+  {{ else if eq .KUBE_NAMESPACE .BRANCH_ENV}}
```

Comment:
this else if statement will never hit because of the if statement on line 5, I think you would want to remove lines 6-7

Reply:
Removing for now but will bring that back on the next ticket for stg and prod